foyer-storage/src/storage.rs

```diff
@@ -135,10 +135,10 @@ where
         Q: Hash + Eq + ?Sized;
 
     #[must_use]
-    fn get<Q>(&self, key: &Q) -> impl Future<Output = Result<Option<CachedEntry<K, V>>>> + Send
+    fn get<Q>(&self, key: &Q) -> impl Future<Output = Result<Option<CachedEntry<K, V>>>> + Send + 'static
```

Comment:
`get` will be spawned on seperate runtime so it's fine for us to make it static instead. In this way, we remove the reference of `&Q` entirely and no clone anymore.